src/main/java/pages/ResizablePage.java

```diff
@@ -22,38 +27,31 @@ public ResizablePage(WebDriver driver) {
     public void assertThatSizeOfElementWasChanged() {
         WebElement elementAfterResize = getDriver().findElement(By.cssSelector("#resizable"));
         Dimension elementSize = elementAfterResize.getSize();
-        System.out.println(elementSize.getWidth());
-        System.out.println(elementSize.getHeight());
-
+        System.out.println("End width: " + elementSize.getWidth());
+        System.out.println("End height: " + elementSize.getHeight());
         LOG.info("Assert that Wight is the same");
-        Assert.assertEquals(183, elementSize.getWidth());
+        Assert.assertEquals("Width is wrong after resize", (startWidthElement + OFFSET - anchorWidth - 1), elementSize.getWidth());
         LOG.info("Assert that Height is the same");
-        Assert.assertEquals(183, elementSize.getHeight());
+        Assert.assertEquals("Height is wrong after resize",(startHeightElement + OFFSET - anchorHeight - 1), elementSize.getHeight());
     }
 
-
-public ResizablePage resizeElement() {
-
-    WebElement resizeableElement = driver.findElement(By.cssSelector("#resizable > div:nth-of-type(3)"));
-    resize(resizeableElement, 50,50);
-
-    return this;
-}
-
-    public void resize(WebElement elementToResize, int xOffset, int yOffset) {
-
-            Actions action = new Actions(driver);
-            action.clickAndHold(elementToResize).moveByOffset(xOffset, yOffset).release().build().perform();
+    public ResizablePage resizeElement() {
+        WebElement resizeableElement = driver.findElement(By.cssSelector("#resizable"));
```

Comment:
Why You had 16px in here? Because You were trying to calculate the size of anchor element, not the full div container.